src/VS.Web.CG.Mvc/Identity/identitygeneratorfilesconfig.json

```diff
@@ -511,6 +503,295 @@
         "IsTemplate": true,
         "ShowInListFiles": false
       }
+    ],
+    "WwwRoot": [
+      {
+        "Name": "Site.css",
+        "SourcePath": "wwwroot/css/site.css",
+        "OutputPath": "wwwroot/css/site.css",
+        "IsTemplate": false,
+        "ShowInListFiles": false
+      },
+      {
+        "Name": "Site.min.css",
+        "SourcePath": "wwwroot/css/site.min.css",
+        "OutputPath": "wwwroot/css/site.min.css",
+        "IsTemplate": false,
+        "ShowInListFiles": false
+      },
+      {
+        "Name": "Banner1.svg",
+        "SourcePath": "wwwroot/images/banner1.svg",
+        "OutputPath": "wwwroot/images/banner1.svg",
+        "IsTemplate": false,
+        "ShowInListFiles": false
+      },
+      {
+        "Name": "Banner2.svg",
+        "SourcePath": "wwwroot/images/banner2.svg",
+        "OutputPath": "wwwroot/images/banner2.svg",
+        "IsTemplate": false,
+        "ShowInListFiles": false
+      },
+      {
+        "Name": "Banner3.svg",
+        "SourcePath": "wwwroot/images/banner3.svg",
+        "OutputPath": "wwwroot/images/banner3.svg",
+        "IsTemplate": false,
+        "ShowInListFiles": false
+      },
+      {
```

Comment:
Site.js and Site.min.js I don't think you need (I believe they are empty, you might need to remove the reference from _Layout.cshtml or similar if there is any)

Reply:
Site.js contains some comments, site.js.min is empty.

I think we should generate those files anyway, to have parity with templates.

Reply:
It's not a big deal, so I'm ok with it.

Reply:
If you'd like me to remove it, I will. I've got a couple other changes to commit to this PR - it's not finalized yet.